.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: java
 jdk:
-  - oraclejdk8
+- oraclejdk8
 addons:
   sonarcloud:
-    organization: "improbable-keanu"
-script:
-  - ./gradlew check
-  - 'if [[ ! -z $SONAR_TOKEN ]]; then sonar-scanner; fi'
+    organization: improbable-keanu
+script: deployment/buildAndDeploy.sh
+before_install:
+- openssl aes-256-cbc -K $encrypted_eec04b57ffc0_key -iv $encrypted_eec04b57ffc0_iv
+  -in deployment/secret-keys-keanu.gpg.enc -out deployment/secret-keys-keanu.gpg -d
```

Comment:
It's probably worth putting `**/*.gpg` in the `.gitignore`, even though it's unlikely (impossible?) that someone would generate it locally and accidently commit it.

Reply:
Worth adding just in case - added.